libs/blocks/icon-block/icon-block.css

```diff
@@ -187,8 +187,7 @@
 }
 
 .icon-block.inline .foreground .text-content .icon-area {
-  width: fit-content;
-  display: contents;
+  width: var(--icon-size-m);
```

Comment:
Is it fine to always have a fixed size, are there any use-cases where the icons should have a different size?

Reply:
I agree that this might not be the full answer. If an `l` size icon is used, there might be some unexpected results. I see a lot of rules that set `width`, `min-width`, `max-width` and their `height` equivalents for the `.icon-area` element. An `l` size icon would have a `max-width` of `234px`, a `min-width` of `56px` and a `width` of `40px`, these should probably be consolidated better.

Reply:
> Is it fine to always have a fixed size (also on other use-cases/other consumers), are there any use-cases where the icons should have a different size?



> I agree that this might not be the full answer. If an `l` size icon is used, there might be some unexpected results. I see a lot of rules that set `width`, `min-width`, `max-width` and their `height` equivalents for the `.icon-area` element. An `l` size icon would have a `max-width` of `234px`, a `min-width` of `56px` and a `width` of `40px`, these should probably be consolidated better.

@mokimo, @overmyheadandbody I looked into this point and I realised that `width: min-content` could be used here if we don't want the height to be fixed. Do let me know if this works.